modules/schematics/package.json

```diff
@@ -24,9 +24,5 @@
   "ng-update": {
     "packageGroup": "NG_UPDATE_PACKAGE_GROUP",
     "migrations": "NG_UPDATE_MIGRATIONS"
-  },
-  "peerDependencies": {
-    "@angular-devkit/core": "NG_DEVKIT_VERSION",
```

Comment:
I think this is the only place where we were using `NG_DEVKIT_VERSION`, thus we could also remove it from `defaults.bzl`.